operator_ui/src/containers/Dashboards/Index.js

```diff
@@ -60,6 +65,7 @@ export const Index = useHooks(props => {
 Index.propTypes = {
```

Comment:
Would it be too much to typescript this Index page too?

Reply:
I thought about it, but had converted so many for this story already that it may hide the intent of the story. Happy to do it though if you want